backend/FwLite/MiniLcm.Tests/Validators/EntryValidatorTests.cs

```diff
@@ -11,24 +11,114 @@ public class EntryValidatorTests
         _validator.TestValidate(entry).ShouldNotHaveAnyValidationErrors();
     }
 
     [Fact]
     public void Succeeds_WhenSenseEntryIdMatchesEntry()
     {
-
         var entryId = Guid.NewGuid();
-        var entry = new Entry() { Id = entryId, Senses = [new Sense() { EntryId = entryId, }] };
+        var entry = new Entry() { Id = entryId, LexemeForm = new MultiString(){{"en", "lexeme"}}, Senses = [new Sense() { EntryId = entryId, }] };
         _validator.TestValidate(entry).ShouldNotHaveAnyValidationErrors();
     }
 
     [Fact]
     public void Fails_WhenSenseEntryIdDoesNotMatchEntry()
     {
         var entryId = Guid.NewGuid();
-        var entry = new Entry() { Id = entryId, Senses = [new Sense() { EntryId = Guid.NewGuid(), }] };
+        var entry = new Entry() { Id = entryId, LexemeForm = new MultiString(){{"en", "lexeme"}}, Senses = [new Sense() { EntryId = Guid.NewGuid(), }] };
         _validator.TestValidate(entry).ShouldHaveValidationErrorFor("Senses[0].EntryId");
     }
+
+    [Fact]
+    public void Succeeds_WhenDeletedAtIsNull()
+    {
+        var entryId = Guid.NewGuid();
+        var entry = new Entry() { Id = entryId, DeletedAt = null, LexemeForm = new MultiString(){{"en", "lexeme"}}, Senses = [new Sense() { EntryId = entryId, }] };
+        _validator.TestValidate(entry).ShouldNotHaveAnyValidationErrors();
+    }
+
+    [Fact]
+    public void Fails_WhenDeletedAtIsNotNull()
+    {
+        var entryId = Guid.NewGuid();
+        var now = DateTime.UtcNow;
+        var entry = new Entry() { Id = entryId, DeletedAt = now, LexemeForm = new MultiString(){{"en", "lexeme"}}, Senses = [new Sense() { EntryId = Guid.Empty, }] };
+        _validator.TestValidate(entry).ShouldHaveValidationErrorFor("DeletedAt");
+    }
+
+    [Fact]
+    public void Succeeds_WhenLexemeFormIsPresent()
+    {
+        var entry = new Entry() { Id = Guid.NewGuid(), LexemeForm = new MultiString(){{"en", "lexeme"}} };
+        _validator.TestValidate(entry).ShouldNotHaveAnyValidationErrors();
+    }
+
+    [Fact]
+    public void Fails_WhenLexemeFormIsMissing()
+    {
+        var entry = new Entry() { Id = Guid.NewGuid() };
+        _validator.TestValidate(entry).ShouldHaveValidationErrorFor("LexemeForm");
+    }
+
+    [Fact]
+    public void Fails_WhenLexemeFormHasNoContent()
+    {
+        // Technically the same as Fails_WhenLexemeFormIsMissing -- should we combine them?
+        var entry = new Entry() { Id = Guid.NewGuid(), LexemeForm = new MultiString() };
+        _validator.TestValidate(entry).ShouldHaveValidationErrorFor("LexemeForm");
+    }
+
+    [Fact]
+    public void Fails_WhenLexemeFormHasWsWithEmptyContent()
+    {
+        var entry = new Entry() { Id = Guid.NewGuid(), LexemeForm = new MultiString(){{"en", ""}} };
+        _validator.TestValidate(entry).ShouldHaveValidationErrorFor("LexemeForm");
+    }
+
+    [Theory]
+    [InlineData("CitationForm")]
+    [InlineData("LiteralMeaning")]
+    [InlineData("Note")]
```

Comment:
I like this, it might be more clear to use `nameof(Entry.Note` etc

Reply:
Done.